docs/api-reference/data-fetching/getStaticPaths.md

```diff
@@ -0,0 +1,200 @@
+---
+description: API reference for `getStaticPaths`. Learn how to fetch data and generate static pages with `getStaticPaths`.
+---
+
+# `getStaticPaths`
+
+<details>
+  <summary><b>Version History</b></summary>
+
+| Version  | Changes                                                                                                         |
+| -------- | --------------------------------------------------------------------------------------------------------------- |
+| `v9.5.0` | Stable [Incremental Static Regeneration](/docs/basic-features/data-fetching/incremental-static-regeneration.md) |
+| `v9.3.0` | `getStaticPaths` introduced.                                                                                    |
```

Comment:
Could not format this table properly in the review UI, but a new line should be added
```
| `v10.0.0` | `fallback: 'blocking'` return option added.
```

https://nextjs.org/blog/next-10#blocking-fallback-for-getstaticpaths

Reply:
I created a separate issue for updating these